src/main/java/com/hesoyam/server/controllers/BittrexController.java

```diff
@@ -0,0 +1,70 @@
+package com.hesoyam.server.controllers;
+
+import com.hesoyam.server.models.Bittrex;
+import com.hesoyam.server.repositories.BittrexRepository;
+import org.bson.types.ObjectId;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Sort;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/bittrex")
+public class BittrexController {
+    @Autowired
+    private final BittrexRepository repository;
+
+    public BittrexController(BittrexRepository repository) {
+        this.repository = repository;
+    }
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.GET)
+    Bittrex getByid(@PathVariable("id") ObjectId id)
+    {
+        return repository.findBy_id(id);
+    }
+
+    @RequestMapping(method = RequestMethod.GET)
+    public List<Map<String, Object>> getCurrencyBySymbol(@RequestParam(value="symbol") String symbol)
+    {
+        List<Bittrex> data = repository.findBySymbol(symbol, new Sort("date"));
+        List<Map<String, Object>> response = new ArrayList<>();
+        Map<String, Object>element = new HashMap<>();
+
+        for(Bittrex elem : data)
+        {
+            element.put("d", elem.getDate());
```

Comment:
To taka bzdura która w sumie na nic w kodzie nie wpływa, ale lepiej zmienić kolejność tego na `{d, o, h, l, c}` żeby trzymać się finansowych konwencji.